sources/osg/Polytope.js

```diff
@@ -334,9 +334,9 @@ Polytope.prototype = MACROUTILS.objectInherit( Object.prototype, {
      * calculated elsewhere then use transformProvidingInverse() instead.
      * See http://www.worldserver.com/turk/computergraphics/NormalTransformations.pdf*/
     transform: function ( matrix ) {
-        var inverse = new Matrix();
-        inverse.invert( matrix );
-        this.transformProvidingInverse( inverse );
+        var inv = Matrix.create();
```

Comment:
@jtorresfabra this function was never called I guess (Matrix is not a function)

Not sure if we can re-use matrix or not.